asteroids_code/BoilerPlate/Bullet.cpp

```diff
@@ -0,0 +1,74 @@
+#include "Bullet.hpp"
+//OpenGL include
+#include <GL/glew.h>
+#include <SDL2/SDL_opengl.h>
+
+Bullet::Bullet()
+{
+	origin = Vector2(0.0f, 0.0f);
```

Comment:
You should have a static function in `Vector2` that is called Origin already. In addition, the default constructor should return a vector on the origin.